[2024-10-22T14:56:50.419+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-22T14:56:50.480+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aula_2_4_TwitterDAG.twitter_datascience scheduled__2024-10-20T00:00:00+00:00 [queued]>
[2024-10-22T14:56:50.516+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aula_2_4_TwitterDAG.twitter_datascience scheduled__2024-10-20T00:00:00+00:00 [queued]>
[2024-10-22T14:56:50.518+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-22T14:56:50.553+0000] {taskinstance.py:2330} INFO - Executing <Task(TwitterOperator): twitter_datascience> on 2024-10-20 00:00:00+00:00
[2024-10-22T14:56:50.561+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=324) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-22T14:56:50.564+0000] {standard_task_runner.py:64} INFO - Started process 327 to run task
[2024-10-22T14:56:50.563+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'aula_2_4_TwitterDAG', 'twitter_datascience', 'scheduled__2024-10-20T00:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/aula_2_4_twitter_dag.py', '--cfg-path', '/tmp/tmp3ws15dmz']
[2024-10-22T14:56:50.565+0000] {standard_task_runner.py:91} INFO - Job 252: Subtask twitter_datascience
[2024-10-22T14:56:50.644+0000] {task_command.py:426} INFO - Running <TaskInstance: aula_2_4_TwitterDAG.twitter_datascience scheduled__2024-10-20T00:00:00+00:00 [running]> on host 4e8aee365f90
[2024-10-22T14:56:50.912+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='aula_2_4_TwitterDAG' AIRFLOW_CTX_TASK_ID='twitter_datascience' AIRFLOW_CTX_EXECUTION_DATE='2024-10-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-20T00:00:00+00:00'
[2024-10-22T14:56:50.913+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-22T14:56:51.010+0000] {base.py:84} INFO - Using connection ID 'twitter_default' for task execution.
[2024-10-22T14:56:51.023+0000] {aula_2_2_twitter_hook.py:31} INFO - URL:https://labdados.com//2/tweets/search/recent?query=datascience&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,lang,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2024-10-20T00:00:00.00Z&end_time=2024-10-21T00:00:00.00Z
[2024-10-22T14:56:51.647+0000] {aula_2_2_twitter_hook.py:31} INFO - URL:https://labdados.com//2/tweets/search/recent?query=datascience&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,lang,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2024-10-20T00:00:00.00Z&end_time=2024-10-21T00:00:00.00Z&next_token=1234567890abcdef
[2024-10-22T14:56:51.783+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-22T14:56:51.800+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=aula_2_4_TwitterDAG, task_id=twitter_datascience, run_id=scheduled__2024-10-20T00:00:00+00:00, execution_date=20241020T000000, start_date=20241022T145650, end_date=20241022T145651
[2024-10-22T14:56:51.868+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-10-22T14:56:51.907+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-22T14:56:51.912+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
